docs/components/table.md

```diff
@@ -601,24 +601,24 @@ export default {
 
 ## Events
 
-Bind any DOM [events](https://developer.mozilla.org/en-US/docs/Web/Events) to
-various parts of the table. We support events on both table rows and cells in addition to click elements inside a row (ie. buttons or hyperlinks) without triggering the a row or cell event. You can also add logic to check for `metakey` to support cmd/ctrl when clicking. Examples highlighted below.
+Bind any DOM [events](https://developer.mozilla.org/en-US/docs/Web/Events) to various parts of the table. We support events on both table rows and cells, but must be careful with clickable content in rows when row click is enabled. You can also add logic to check for `metakey` to support cmd/ctrl when clicking. Examples highlighted below.
 
 ### Rows
 
 `@row:<event>` - returns the `Event`, the row item, and the type: `row`
 
 ```vue
-<KTable @row:click="rowHandler" @row:dblclick="rowHandler">
+<KTable @row:click="rowHandler">
 ```
 
+To avoid firing row clicks by accident, the row click handler ignores events coming from `a`, `button`, `input`, and `select` elements. As such click handlers attached to these element types do not require stopping propagation via `@click.stop`.
+
```

Comment:
What if the user still wants the row click to happen even if there is one of these elements that is clicked?

Maybe we change the logic to "if one of those elements AND it does not have a `disabled` attribute"?

Reply:
Good call, will add.

Reply:
Added